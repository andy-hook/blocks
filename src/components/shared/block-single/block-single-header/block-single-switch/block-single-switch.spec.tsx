import React from "react"
import BlockSingleSwitch from "./block-single-switch"
import { renderWithTheme } from "@test-utils"

jest.mock("react-truncate-string")

describe("<BlockSingleSwitch />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <BlockSingleSwitch
        handleDetailsClick={jest.fn}
        handleTransactionsClick={jest.fn}
      />
    )
    expect(tree).toMatchSnapshot()
  })
})
