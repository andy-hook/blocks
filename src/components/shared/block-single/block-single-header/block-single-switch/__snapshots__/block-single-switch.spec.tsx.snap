// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockSingleSwitch /> renders correctly 1`] = `
<section
  className="sc-bdVaJa AMIHk"
>
  <button
    className="sc-bwzfXH uTxCW"
    onClick={[Function]}
  >
    Details
  </button>
  <button
    className="sc-htpNat KtFsv"
    onClick={[Function]}
  >
    Transactions
  </button>
</section>
`;
